spec/index.bs

```diff
@@ -412,6 +412,34 @@ for the [=usage scope=] of this render pass.
 
 Issue(gpuweb/gpuweb#514): Document read-only states for depth views.
 
+## Synchronization ## {#programming-model-synchronization}
+
+For each [=subresource=] of a [=physical resource=], the usage is tracked on the [=Queue timeline=].
+
+Issue: This section will need to be redacted to support multiple queues.
+
+The [=Queue timeline=] could be split into a sequence of [=usage scopes=]:
```

Comment:
Is this supposed to be part of the Issue? If not, then "could" doesn't make sense here

Reply:
no, it's not a part of the issue. I was trying to say that we can consider the queue timeline to be seen as a sequence of usage scopes.

Reply:
Ok, then I think the version I have in the "nits" should be what you intended.